### DISCUSSION TITLE
Welcome to AKILT Discussions!

### DISCUSSION THREAD

**Discussion by Xart3mis** [Xart3mis/AKILT#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by catalcin** [comment]
i hop this project is still alive

**Reply by Xart3mis** [reply]
Sort of... Kind of busy currently, but will be getting back to it soon

**Reply by catalcin** [reply]
I get you
